contracts/scripts/update-round.config.ts

```diff
@@ -0,0 +1,31 @@
+import metadataJson from './grant-round-metadata.json';
+
+type NetworkParams = {
+  roundAddress: string;
+  ipfsRetrievalEndpoint: string;
+  roundLogoPath: string; // this path must be relative to the 'contracts' directory
```

Comment:
wouldn't we just the uploaded image's hash here ?

Reply:
As I think you figured out, the script automates uploading the file to IPFS as well!